generators/chipyard/src/main/scala/TestHarness.scala

```diff
@@ -19,32 +23,96 @@ trait HasTestHarnessFunctions {
 }
 
 trait HasHarnessSignalReferences {
+  // clock/reset of the chiptop reference clock (can be different than the implicit harness clock/reset)
   def harnessClock: Clock
   def harnessReset: Reset
   def dutReset: Reset
   def success: Bool
 }
 
+class HarnessClockInstantiator {
+  private var _clockMap: HashMap[String, (Double, ClockBundle)] = HashMap.empty
```

Comment:
I think in general it is better to use a SortedMap that keeps orderings when building Chisel hardware to keep names stable.
See: https://github.com/chipsalliance/rocket-chip/commit/f86489b59e82c4ae4e39b6ba0caf9fd279d4aa16 for example

I can't quite tell if this will be an issue in this case but I don't think it will hurt.